x/distribution/spec/04_messages.md

```diff
@@ -6,13 +6,16 @@ order: 4
 
 ## MsgSetWithdrawAddress
 
-By default a withdrawal address is delegator address. If a delegator wants to change it's
-withdrawal address it must send `MsgSetWithdrawAddress`.
+By default, the withdraw address is delegator address. If a delegator wants to change its withdraw address it must send `MsgSetWithdrawAddress`.
+This is only possible if the parameter `WithdrawAddrEnabled` is set to `true`.
+
+The withdraw address cannot be any of the module accounts. These are blocked from being withdraw addresses by being added to the distribution keeper's `blockedAddrs` array at initialization.
+
+Response:
 
 +++ https://github.com/cosmos/cosmos-sdk/blob/v0.40.0/proto/cosmos/distribution/v1beta1/tx.proto#L29-L37
```

Comment:
nit, I think we should update these to the latest version

Reply:
hmm, should we do that here? or would you want to script it so that it isn't done in a patchwork fashion?